ext/storage_sources/azure_store/README.md

```diff
@@ -0,0 +1,84 @@
+# WiredTiger's Azure Extension
+## 1. Introduction
+This extension allows WiredTiger storage source extensions to read from and write to objects stored
+in Azure Blob Storage using WiredTiger’s provided internal abstraction for storing data in an object
+storage service.
+
+## 2. Building and running
+This section describes how to build WiredTiger with the Azure extension enabled.
+
+### Requirements
+* CMake 3.13 or higher
+* G++ 8.4 or higher
+* clang version 6.0.0
+
+### Building
+
+There is currently only 1 way to build WiredTiger with Azure extension:
+1. Letting CMake manage the Azure SDK dependency as an external project, letting it download, link
+    and build the extension.
+
+There are two CMake flags associated with the Azure extension: `ENABLE_AZURE` and `IMPORT_AZURE_SDK`.
+* `ENABLE_AZURE=1` is required to build the Azure extension.
+* `IMPORT_AZURE_SDK={external}` is used to set the build method.
+    *   `external` tells the compiler to search for an existing system installation of the SDK.
+    *    This flag should be set alongside the `ENABLE_AZURE` flag.
+    *    If the `IMPORT_AZURE_SDK` flag is not specified, the compiler will assume a system
+            installation of the SDK which will be implemented at a later stage.
+
+### Letting CMake manage the SDK dependency as an external project
+
+This method configures CMake to download, compile, and install the Azure SDK while building the
+Azure extension.
+
+```bash
+# Create a new directory to run the build from
+mkdir build && cd build
+
+# Configure and run cmake with Ninja
+cmake -DENABLE_PYTHON=1 -DHAVE_UNITTEST=1 -DIMPORT_AZURE_SDK=external -DENABLE_AZURE=1 -G Ninja ../.
+ninja
+```
+
+* The compiler flag `IMPORT_AZURE_SDK` must be set to `external` for this build method.
+* `ENABLE_AZURE` defaults to looking for a local version, the `IMPORT_AZURE_SDK` setting will
```

Comment:
I don't agree that it works like this. IMPORT_AZURE_SDK must be set period if used with ENABLE_AZURE. If not it should be erroring out.

Reply:
sure i've updated the comment to reflect that `IMPORT_AZURE_SDK` must be set if used with `ENABLE_AZURE` if not it will error out. https://github.com/wiredtiger/wiredtiger/pull/8802/commits/1329d79760b71f8779343c5eee6b23f5933dced0